specs/olm.spec

```diff
@@ -48,4 +48,23 @@ Importance: Critical
 Uninstalls `openshift-pipelines` operator using olm
 Steps:
   * Validate Operator should be installed
-  * Uninstall Operator
\ No newline at end of file
+  * Uninstall Operator
+
+##Check server side components versions
+Tags: install, upgrade
+Steps:
+    * Check version of component "pipeline"
+    * Check version of component "triggers"
+    * Check version of component "operator"
+    * Check version of component "pipelines-as-code"
+    * Check version of "OSP"
+
+##Check client versions
+Tags: install, upgrade
+Steps: 
+    * Download and extract CLI from cluster
+    * Check "Client" version in "tkn"
+    * Check "PAC" version in "tkn"
+    * Check "Client" version in "opc"
+    * Check "Tekton" version in "opc"
+    * Check "PAC" version in "opc"
```

Comment:
These steps don't read very naturally. Wouldn't it be more readable and easier to implement if there was simply something like
1. `Check "tkn" client version`
2. `Check "tkn-pac" client version`
3. `Check "opc" client version`